Podfile

```diff
@@ -4,7 +4,7 @@ use_frameworks!
 def shared_pods
   pod 'Pelias', '~> 1.0.0-beta'
   pod 'OnTheRoad', :git => 'https://github.com/mapzen/on-the-road_ios.git', :commit => '603fe7a'
-  pod 'Tangram-es', :git => 'https://github.com/tangrams/ios-framework.git', :branch => 'universal-0.5.1-preview' 
+  pod 'Tangram-es', :git => 'https://github.com/tangrams/ios-framework.git', :branch => 'master', :commit => 'a1b1338'
```

Comment:
Looks like the 0.5.1 release has fully propagated in Cocoapods. Lets use that instead?